diff --git a/lib/rsa.js b/lib/rsa.js
--- a/lib/rsa.js
+++ b/lib/rsa.js
@@ -1175,5 +1175,8 @@
           var capture = {};
           var errors = [];
           if(!asn1.validate(obj, digestInfoValidator, capture, errors) ||
-            obj.value.length !== 2) {
+            obj.value.length !== 2 ||
+            // asn1.validate permits extra nested children (CVE-2026-85393).
+            obj.value[0].value.length !==
+              (('parameters' in capture) ? 2 : 1)) {
             var error = new Error(
